Improve various messages for the dotnet-dev-certs https command

0xced opened the pull request:
* Show an error if importing a certificate fails because of an exception
* Improve the output message when running dotnet-dev-certs https --check
* Improve the --trust option documentation (when combined with --check)
* Show _which_ certificates are valid or trusted

/cc @javiercn

0xced committed:
Show an error if importing a certificate fails because of an exception

0xced committed:
Improve the output message when running dotnet-dev-certs https --check

Show the message `A valid certificate was found.` as output instead of verbose and also suggest to run the command with both --check and --trust options to ensure that the certificate is not only valid but also trusted.

javiercn commented:
@0xced does https://github.com/dotnet/aspnetcore/pull/25383/ make this PR obsolete?

0xced commented:
No, they are two distinct pull requests.

0xced committed:
Remove dead code

0xced committed:
Improve the --trust option documentation (when combined with --check)

Explicitly state that when combined with the --check option, --trust is used to validate that the certificate is trusted.

0xced committed:
Show _which_ certificate is valid or trusted

0xced committed:
Show _all_ valid or trusted certificates found.

0xced commented:
I just rebased on the `main` branch and resolved all conflicts. Could you please have a look, @javiercn?

javiercn reviewed (approved):
Thanks for the contribution!
